tests/func/test_run_single_stage.py

```diff
@@ -91,7 +91,7 @@ def test(self):
 
         with self.assertRaises(DependencyDoesNotExistError):
             self.dvc.run(
-                cmd="",
+                cmd="command",
```

Comment:
A lot of tests here were `cmd` is never run (they eventually raise Exception before running the command), I have used `command`.